src/contrib/cblas/cblas.cc

```diff
@@ -67,9 +74,12 @@ struct CblasSgemmBatchOp {
     CBLAS_TRANSPOSE trans_a = BooleanToTranspose(ta);
     CBLAS_TRANSPOSE trans_b = BooleanToTranspose(tb);
 #if USE_MKL_BLAS == 1
-    std::vector<const float*> A_array(batch_size);
-    std::vector<const float*> B_array(batch_size);
-    std::vector<float*> C_array(batch_size);
+    static std::vector<const float*> A_array;
```

Comment:
`static` is not thread safe here.

Reply:
AFAIK, TVM runs all ops sequentially, so two calls to BatchMatMul cannot happen in parallel. What do you think?

Reply:
There is usecase that upper layer service launches multiple TVM runtime, each itself is single threaded on one thread.